adcc/backends/pyscf.py

```diff
@@ -210,6 +268,23 @@ def get_restricted(self):
                                    "not determine restricted / unrestricted.")
         return restricted
 
+    def get_n_atoms(self):
+        n_atoms = self.scfres.mol.natm
+        return n_atoms
+
+    def get_nuclear_charges(self):
+        charges = self.scfres.mol.atom_charges()
+        return charges
+
+    def get_nuclear_masses(self):
+        masses = self.scfres.mol.atom_mass_list()
+        return masses
+
+    def get_coordinates(self):
+        coords = self.scfres.mol.atom_coords()
+        coords = np.reshape(coords, (3 * coords.shape[0]))
+        return coords
```

Comment:
Why do the coordinates/other properties above need to be exposed?

Reply:
The idea was to be able to compute nuclear momets (electric quadrupole & electric quadrupole traceless) with different gauge origins in responsefun.

Reply:
In principle, a selection of different gauge origins should be possible within the `get_nuclear_multipole` method of the `HartreeFockProvider` class. However, this is a little more complicated here than for the operator integrals because also `libadcc` would need to be changed for this and we only need it for multipoles with an order > 1. Then the coordinates etc. would not need to be exposed, which would definitely make more sense. @maxscheurer do you have an idea how we could handle this?

Reply:
These new properties can then be deleted.